### PR TITLE
Sort works: sort the work list in a collection page

### PR DESCRIPTION
**Pull request by MPLSFedResearchTZ**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -73,9 +73,11 @@ group :development, :test do
   gem 'bummr'
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'capybara'
+  gem 'debase', '~> 0.2.4.1'
   gem 'fcrepo_wrapper'
   gem 'rspec-rails'
   gem 'rspec_junit_formatter'
+  gem 'ruby-debug-ide'
```
Comment (mark-dce) [inline Gemfile:80]: This gem, generally all debugging and testing gems, should not be run in the production environment, please move the testing gems to the "development, testing" gem group lower in this file.

**Review by mark-dce — CHANGES_REQUESTED** [review]

**Comment by MPLSFedResearchTZ** [comment]
The rubyu-debug-ide gem is in the "development, testing" group as shown below:
image

File: Gemfile (hunk 1 above)
Comment (MPLSFedResearchTZ) [reply Gemfile:80]: The rubyu-debug-ide gem is in the "development, testing" group as shown below:
image

Comment (mark-dce) [reply Gemfile:80]: Tao - my apologies, I was misreading the file because of the way git was displaying the diff.  I'm sorry for the delay.

**Review by mark-dce — APPROVED** [review]

**Pull request merged**